app.js

```diff
@@ -55,7 +55,7 @@ axios
         clientID: CLIENT_ID,
         clientSecret: CLIENT_SECRET,
         callbackURL: 'http://localhost:3000/authorization-code/callback',
-        scope: 'groups profile offline_access',
```

Comment:
I do not know why removing groups would be necessary to make 3pcd work

Reply:
It's not necessary for the third-party cookies sample effort. The groups scope is not necessary for this app and is confusing.

I want to set a baseline for subsequent work, and I don't want groups' scope to be added everywhere. And will resolve #6 as a bonus.